tools/delete_timepoint.php

```diff
@@ -273,7 +284,7 @@ function deleteTimepoint($CandID, $sessionID, $confirm, $printToSQL, $DB, $outpu
         // Delete from session
         echo "\n-- Deleting from session.\n";
         $DB->delete('session', array('ID' => $sessionID));
-    } elseif ($printToSQL) {
+    } else {
```

Comment:
Yes, (very) little time waisted if someone call the delete_timepoint.php script directly without tosql nor confirm. The check for tosql is at the end of this block. Si if someone directly called delete_timepoint.php with tosql, it will print as before but if it was called from inside the delete_candidate.php script, output will be returned to the calling script to include at the right place and not in a separate script.